### ISSUE TITLE
com.tomirio.schakert.perft.PerftTest

### ISSUE THREAD

**Issue by pawc** [tomirio619/Schakert#1]
tests stuck after "Results of divide perft at depth 3"
Not sure how to proceed

**Comment by tomirio619** [comment]
Hey pawc, I assume you are talking about the test case in the test function `Position6()`.
It involves the following FEN string: 

_r3k2r/p1ppqpb1/bn2pnp1/3PN3/1p2P3/2N2Q1p/PPPBBPPP/R3K2R w QKqk -_. 

Although the test case takes a long time to finish at depth 4, it does return a result. 
I've also added a method that outputs the time it took to get the results.
I will push it today.
On my PC, this partical FEN at depth 4 took ~3 minutes to finish.
This also applies to divide perft with the initial FEN string at depth 5 (I also removed depth 6 in the test case).

I hope this helps!


If you have any more questions, don't hesitate to let me know!
King regards,
Tom
